README.md

```diff
@@ -1,37 +1,98 @@
-# National Document Repository Infrastructure 
+# National Document Repository Infrastructure
 
+## Prerequisites
 
-
-## Prerequisite
 Ensure the following Prereqs are installed first (can use brew on Mac/Linux or Chocolatey on Windows)
-- [Terraform Docs](https://terraform-docs.io/) - for formmating terraform documentation
-```bash
-brew install terraform-docs
-```
+
+- [Terraform Docs](https://terraform-docs.io/) - for formatting Terraform documentation
+
+    ```bash
+    brew install terraform-docs
+    ```
+
 - [findutils](https://www.gnu.org/software/findutils/) - Needed for scripts running on MacOSX
-```bash
-brew install findutils
-```
+
+    ```bash
+    brew install findutils
+    ```
 
 ## Repository best practices
 
-We provide a makefile to ensure consistency and provide simplicity. It is strongly advised, both when planning and applying terraform, that this is done via the makefile.
+We provide a `makefile` to ensure consistency and provide simplicity. It is strongly advised, both when planning and applying Terraform, that this is done via the `makefile`.
 
-The `make pre-commit` command this will format all terraform code, and re-create all README.md files. This should be run before every commit to keep the code base clean.
+The `make pre-commit` command will format all Terraform code, and re-create all `README.md` files. This should be run before every commit to keep the code base clean.
 
 ## Using Workspaces
-To initialise the S3 backend, cd to infrastructure folder and run 
+
+To initialise the S3 backend:
+
 ```bash
+cd infrastructure/
 terraform init -backend-config=backend.conf
 ```
 
 ### The makefile
 
-The following commands currently exist in the make file:
+The following commands currently exist in the `makefile`:
 
 - `make pre-commit` -> runs both the `make generate-docs` and `make format-all` commands.
 
-
 ### Deploying to a new AWS Account
 
-The details on how to run this terraform process on a new AWS account can be found on our confluence guides found [here](https://gpitbjss.atlassian.net/wiki/spaces/TW/pages/12581568619/Infrastructure+-+Deploy+to+a+new+Account)
\ No newline at end of file
+The details on how to run this Terraform process on a new AWS account can be found on our [Confluence guide](https://gpitbjss.atlassian.net/wiki/spaces/TW/pages/12581568619/Infrastructure+-+Deploy+to+a+new+Account).
+
+## Troubleshooting
+
+### Resolving "Error: Error acquiring the state lock" for Sandboxes
+
+If a manual or CRON destroy fails and results in a state lock, follow these steps to resolve the issue:
+
+#### Step 1: Check for Recovery Points
+
+1. Log in to the AWS console.
+1. Navigate to `AWS Backup -> Vaults`.
+1. Confirm there are no `Recovery points` associated with the sandbox environment you're trying to destroy.
+1. If any recovery points exist, manually delete them.  
+    > [!WARNING]
```

Comment:
Can the icon for warning be used instead of the word?
I see that the [!important] rendered nicely but not the [!warning part]